### PR TITLE
Removed tabConnection from ChromeProxyService.

### PR DESCRIPTION
**Pull request by pisimulation**

### CODE REVIEW

File: dwds/lib/src/dwds_vm_client.dart [hunk 1]
```diff
@@ -59,17 +59,18 @@ class DwdsVmClient {
     await client.registerService('hotRestart', 'DWDS fullReload');
 
     client.registerServiceCallback('fullReload', (_) async {
-      await chromeProxyService.tabConnection.page.enable();
+      await chromeProxyService.wipDebugger.connection.page.enable();
       // TODO: use built in `page.reload` once it works,
       // https://github.com/google/webkit_inspection_protocol.dart/issues/44
-      await chromeProxyService.tabConnection.sendCommand('Page.reload');
+      await chromeProxyService.wipDebugger.connection
```
Comment (grouma) [inline dwds/lib/src/dwds_vm_client.dart]: Can you just sendCommand on the wipDebugger? Does it have to be on the connection?

File: dwds/lib/src/dwds_vm_client.dart [hunk 2]
```diff
@@ -59,17 +59,18 @@ class DwdsVmClient {
...
+          .sendCommand('Page.reload');
       return {'result': Success().toJson()};
     });
     await client.registerService('fullReload', 'DWDS');
 
     client.registerServiceCallback('ext.dwds.screenshot', (_) async {
-      await chromeProxyService.tabConnection.page.enable();
-      var response = await chromeProxyService.tabConnection.page
+      await chromeProxyService.wipDebugger.connection.page.enable();
+      var response = await chromeProxyService.wipDebugger.connection.page
```
Comment (grouma) [inline dwds/lib/src/dwds_vm_client.dart]: Same here

File: dwds/lib/src/handlers/dev_handler.dart [hunk 3]
```diff
@@ -173,7 +173,7 @@ class DevHandler {
             serializers.serialize(DevToolsResponse((b) => b..success = true))));
 
         appServices.connectedInstanceId = message.instanceId;
-        await appServices.chromeProxyService.tabConnection
+        await appServices.chromeProxyService.wipDebugger.connection
```
Comment (grouma) [inline dwds/lib/src/handlers/dev_handler.dart]: Same here

File: dwds/lib/src/dwds_vm_client.dart (hunk 1 above)
Comment (pisimulation) [reply dwds/lib/src/dwds_vm_client.dart]: Yes, we can. It actually doesn't have to be on the connection.

File: dwds/lib/src/connections/debug_connection.dart [hunk 4]
```diff
@@ -17,7 +17,8 @@ class DebugConnection {
   final _onDoneCompleter = Completer();
 
   DebugConnection(this._appDebugServices) {
-    _appDebugServices.chromeProxyService.tabConnection.onClose.first.then((_) {
+    _appDebugServices.chromeProxyService.wipDebugger.connection.onClose.first
+        .then((_) {
```
Comment (natebosch) [inline dwds/lib/src/connections/debug_connection.dart]: [optional] consider using [`whenComplete`](https://api.dart.dev/stable/2.4.0/dart-async/Future/whenComplete.html)

**Pull request merged**